packages/ui-stencil/src/components/internal/orama-chat-messages-container/orama-chat-assistent-message/orama-sources/orama-sources.tsx

```diff
@@ -173,9 +185,10 @@ export class OramaSources {
                   // TODO: Use a URL object instead
                   href={`${this.sourceBaseURL}${source[this.sourcesMap.path]}`}
                   class="source"
-                  target="_blank"
-                  rel="noopener noreferrer"
+                  target={this.linksTarget}
+                  rel={this.linksRel}
                   id={`source-${index}`}
+                  onClick={() => this.handleItemClick(source)}
```

Comment:
Ideally I think it would be better to replace <a> with <button> and allow the users more flexible configuration, this is somehow related to this linear here: https://linear.app/oramasearch/issue/ORM-1673/add-onitemclick-prop. Keeping a note as reminder, since ORM-1673 will likley affect also the new linksTarget and linksRel prop, and we will no longer need those